src/python/grpcio/grpc/__init__.py

```diff
@@ -1217,8 +1219,8 @@ def abort(self, code, details):
             termination of the RPC.
 
         Raises:
-          Exception: An exception is always raised to signal the abortion the
-            RPC to the gRPC runtime.
+          AbortError: An grpc.AbortError is always raised to signal the abortion
```

Comment:
Nit: "an" -> "a". Here and throughout. Not usually important, but this will end up in the API docs.

Reply:
Right, better fix this.